### ISSUE TITLE
UI Updates

### ISSUE THREAD

**Issue by seanmobrien** [seanmobrien/we-dont-need-no-education#59]
The colorful theme you built is beautiful; however, the following factors limit it's effectiveness - 

1) the background is still dark/black, which prevents the colors from really being seen and put us out of accessibility compliance
2) the button to activate it is only available on a single page; it should be available on all pages in the project

Please address these issues so the world can see your creativity for the beautiful thing that it is :)
